### PR TITLE
Clarified IcoMoon icon updating steps

### PR DESCRIPTION
**Pull request by ktuite**
I added more details to this part of the docs based on where I got confused. I also made some pictures, but I'm not sure where they should go or if they are necessary with more explicit instructions. 

icomoon-choose-new-icons

icomoon-font-awesome

### CODE REVIEW

**Review by matthew-white — COMMENTED** [review]
Thanks for adding this! I think this helps a lot.

> I also made some pictures, but I'm not sure where they should go or if they are necessary with more explicit instructions.

I do think things are clearer with the updated instructions, but I also think the pictures help. Feel free to add them! Though in terms of how to add them, I'm actually not sure the best way to do that. One option might be to use the image URLs from the PR description above.

**Comment by lognaturel** [comment]
> One option might be to use the image URLs from the PR description above.

That makes me nervous. Seems unexpected that if for some reason the PR description is edited the contributing guide would break! I'd create a `docs` folder to put the contribution guide in and put images there (https://docs.github.com/en/github/creating-cloning-and-archiving-repositories/about-readmes#relative-links-and-image-paths-in-readme-files).

**Comment by matthew-white** [comment]
A `docs/` directory (and maybe a `docs/img/` subdirectory?) sounds good to me. :+1: I think I'd lean toward keeping `CONTRIBUTING.md` where it is, in part because we link to it from the `central` repository. (The Briefcase repository similarly also has a root `CONTRIBUTING.md` along with a separate `docs/` directory.) But also happy to move it if that's your preference!

**Comment by matthew-white** [comment]
Ah sorry, closed this accidentally, I think things should be OK now! 😅

**Comment by lognaturel** [comment]
No pref on my end for moving the markdown docs. The only thing I really care about is explicitly putting the image artifacts in the repo.

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -252,15 +252,17 @@ Instead of scoped CSS, use `id` and `class` attributes to style components:
...
 1. Click the "Import Icons" button in IcoMoon. Select [`icomoon.json`](/icomoon.json). When prompted, load the settings stored in the file.
-2. Add Font Awesome.
-3. Move the imported set above Font Awesome. (This should help preserve the icon order and minimize the diff.)
-4. Update the icons.
-5. Download the new font, then copy the files into the repository. You will need to update the paths in the CSS, and you may need to prettify the JSON file.
+2. Scroll down to the "Add Icons From Library" link and add Font Awesome.
+3. Move the imported set above Font Awesome, using the 3-bar icon to the right of the imported set's title. (This should help preserve the icon order and minimize the diff.)
+4. Update the icons by selecting (highlighting) the new icons to add. They don't need to be moved or altered.
+5. Download the new font, then copy the files (`icomoon.css`, `fonts/*`, `icomoon.json`) into their locations in the repository.
+	* You will need to rename the files and update the paths in the CSS (`fonts/icomoon.ttf?...` becomes `/fonts/icomoon.ttf?...` with a beginning slash).
```
[the remark this answers is not in the document]
Comment (ktuite) [reply CONTRIBUTING.md:264]: Will do!

**Review by matthew-white — APPROVED** [review]
Very helpful! 👍

**Pull request merged**